docs/change_log/2022_06_bot_permissions.mdx

```diff
@@ -0,0 +1,24 @@
+---
+title: Changes to Bot Permissions for Interactions and Webhooks
+date: 2022-06-29
+breaking: true
+---
+
+#### Upcoming Changes
+
+> warn
+> Starting on **September 12, 2022**, apps that aren’t using the new `resume_gateway_url` field to resume gateway sessions will be disconnected significantly faster than normal.
```

Comment:
This is the warning box from [docs/change_log/2022_08_gateway_resume.mdx](https://github.com/discord/discord-api-docs/pull/5585/files#diff-00e93ab4f941bcef17595e69f04535687ce2ab838fd989ae4136a9ec77097db8); the original content is nowhere to be found. The correct content should be [this](https://discord.com/developers/docs/change-log#changes-to-bot-permissions-for-interactions-and-webhooks).